### PR TITLE
[Fix](planner)fix select from inline table return only the first row

### PR DESCRIPTION
**Pull request by sohardforaname**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

File: fe/fe-core/src/main/java/org/apache/doris/analysis/SelectStmt.java [hunk 1]
```diff
@@ -589,7 +589,7 @@ public void analyze(Analyzer analyzer) throws UserException {
                 } else {
                     resultExprs.add(rewriteQueryExprByMvColumnExpr(expr, analyzer));
                 }
-                colLabels.add(expr.toColumnLabel());
+                colLabels.add("col_" + colLabels.size());
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/SelectStmt.java:592]: why change this?

Comment (sohardforaname) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/SelectStmt.java:592]: it's according to trino.

**Comment by sohardforaname** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**